mongodb_consistent_backup/Common/LocalCommand.py

```diff
@@ -36,7 +43,10 @@ def parse_output(self):
 
     def run(self):
         try:
-            self._process = Popen(self.command_line, stdout=PIPE, stderr=PIPE)
+            print("cmd : ")
+            print(" ".join( self.admin_command_line+ ["&&"]+ self.command_line))
```

Comment:
Should we use logging.debug here instead of print? It also looks like we gate that sort of output behind self.verbose elsewhere in this file.